code/modules/clothing/clothing.dm

```diff
@@ -477,6 +477,13 @@ BLIND     // can't see anything
 			. += mutable_appearance('icons/effects/item_damage.dmi', "damaged[blood_overlay_type]")
 		if(blood_DNA)
 			. += mutable_appearance('icons/effects/blood.dmi', "[blood_overlay_type]blood")
+		var/mob/living/carbon/human/M = loc

```

Comment:
Okay, this seems a bit convoluted. I tried doing it how you explained, but all I managed to do was break accessories showing up at all.